ads/google/a4a/test/test-google-data-reporter.js

```diff
@@ -192,7 +198,7 @@ describe('#getLifecycleReporter', () => {
         fakeElt.setAttribute('data-amp-slot-index', '22');
         fakeElt.setAttribute('type', 'doubleclick');
         fakeElt.setAttribute(EXPERIMENT_ATTRIBUTE,
-            DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES.experiment);
+            DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH.experiment);
```

Comment:
!! Did not duplicate experiment with DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH. Ditto line 233. Happy to do so, just unsure if that is needed for this to push.